packages/vscode-ui5-language-assistant/test/suite/completions/completion-spec.ts

```diff
@@ -1,30 +1,35 @@
 import * as vscode from "vscode";
-import { expect } from "chai";
 import { resolve, dirname } from "path";
 import { map } from "lodash";
-import { promises as fs } from "fs";
+import { expect } from "chai";
 import { TextDocument, Position } from "vscode-languageserver";
-import { deactivate } from "../../src/extension";
+import { sleep, setFileTextContents } from "../test-utils";
+
+const EXTENSION_START_TIMEOUT = 5000;
+
+const pkgJsonPath = resolve(__dirname, "..", "..", "..", "..", "package.json");
 
-const pkgJsonPath = require.resolve(
-  "vscode-ui5-language-assistant/package.json"
-);
 const rootPkgFolder = dirname(pkgJsonPath);
+const scenarioPath = resolve(
```

Comment:
you don't have to go all the way up to rootPkg and then down.
Just go up to the first shared parent folder of the suites and the fixtures

Reply:
we don't go all way up to rootPkg.
CWD here is the root package folder, so it's better to explicitly use it